plenum/common/messages/fields.py

```diff
@@ -181,6 +188,14 @@ def _specific_validation(self, val):
                 return val_error
 
 
+class HetroMapField(FieldBase):
```

Comment:
Hetro is for Heterogeneous. It means the keys and value can be of arbitrary and distinct types